ovos_gui_plugin_shell_companion/__init__.py

```diff
@@ -139,11 +137,11 @@ def handle_device_display_factory(self, message):
         self.gui.show_page("SYSTEM_AdditionalSettings.qml", override_idle=True)
 
     def handle_device_display_settings(self, message):
-        LOG.debug(f"Display settings: {self.local_display_config}")
+        LOG.debug(f"Display settings: {self.config}")
         self.gui['state'] = 'settings/display_settings'
         # wallpaper_rotation data is determined via Messagebus in Qt directly
-        self.gui['display_auto_dim'] = self.local_display_config.get("auto_dim", False)
-        self.gui['display_auto_nightmode'] = self.local_display_config.get("auto_nightmode", False)
+        self.gui['display_auto_dim'] = self.config.get("auto_dim", False)
+        self.gui['display_auto_nightmode'] = self.config.get("auto_nightmode", False)
```

Comment:
Consider defaulting to True. It's new behavior but you mentioned people have been complaining about the assistant being too bright at night. It's not unreasonable to default to dimming at night.